spec/models/spree/line_item_spec.rb

```diff
@@ -68,7 +68,8 @@ module Spree
       let!(:li) { create(:line_item, variant: v, quantity: 10, max_quantity: 10) }
 
       before do
-        v.update_attributes! on_hand: 5
+        v.on_hand = 5
+        v.save
```

Comment:
@sauloperez this simple test only works if I save the variant after setting on_hand on it (uses VariantStock). I am not sure if we should keep this save here in the spec on in the VariantStock methods themselves.

Reply:
`update_attributes!` requires [attr_accessible](https://apidock.com/rails/ActiveRecord/Base/attr_accessible/class) to work. I went ahead and commit a fix so that you see it in action. Do as you please with it.

We can fix the other attributes of VariantStock either here, in another PR or create a separate issue. Your call.

Reply:
ok, if you think this update_attributes thing with attr_acessible is the best way to go here I can do it in this PR yes. thanks!
I wouldnt do for the other methods if I dont need them, yagni.

Reply:
Good point. Let's see them failing first :ok_hand:

Reply:
Good point. Let's see them failing first :ok_hand:

Reply:
done, thanks for one more lesson!